pkg/virtctl/memorydump/memorydump.go

```diff
@@ -262,7 +284,71 @@ func getMemoryDump(namespace, vmName string, virtClient kubecli.KubevirtClient)
 		}
 	}
 
-	return createMemoryDump(namespace, vmName, claimName, virtClient)
+	if err := createMemoryDump(namespace, vmName, claimName, virtClient); err != nil {
+		return err
+	}
+
+	if outputFile != "" {
```

Comment:
Don't we need to return an error if the outputFile is empty and we want to perform the download?

Reply:
if we use the get sub-command then we know if the user wants to download by adding the outputFile flag. If we use the download sub-command then we do make sure that outputFile is not empty.

Reply:
I see, you are setting a default value `output`, so it is never empty

Reply:
no this is the name of the flag, the default value is "":
cmd.Flags().StringVar(&outputFile, "output", "", "Specifies the output path of the memory dump to be downloaded.")

Reply:
I didnt want `create` since its not `creating` a memory dump, its getting a memory dump, also there is the create flag for creating a pvc for the dump or using existing one..
you thing doing `virtctl memory-dump dump` is better then `virtctl memory-dump get`? I dont know..

Reply:
I agree that `get` may be a little confusing. Maybe you can rename `get` to `create` and use it exclusively for creating the memory dump, and then modify the `download` function to do the whole process if certain flags are specified.

Reply:
That is what I did at first - having download do all process but I was told that it is confusing, that download should only download existing memory dump. so I added an output flag to the get so you can get the memory dump to an output file not only to the pvc.
At first I used create and in the PR that I introduced it I was suggested to change to get. This PR just moved the code to a new file, the command is `get` a while now..

Reply:
Okay, thanks for the history on this. Like I said I was very confused by the get not actually getting output to my local machine but instead saving the memory dump to the specified PVC. And from what @alicefr wrote I think she thought the same.